src/http.js

```diff
@@ -1,4 +1,5 @@
-import fetch from 'isomorphic-fetch'
+import 'es6-promise/auto'
```

Comment:
This is recommended by the maintainer of `isomorphic-fetch`, this provide a polyfill for es6 promises.
But i dont think it is really really important, because many browser have implemented the ES6 Promises
- https://kangax.github.io/compat-table/es6/#test-Promise
- https://caniuse.com/#feat=promises
So i remove the import and dependency :)